docs/tutorial/image_det.md

```diff
@@ -1,3 +1,128 @@
 # Training for Object Detection
 
-Object detection is in preparation.
+This guide trains a neural network model to object detection of Human Face and Human Hand in the [Open Images Dataset V4](https://storage.googleapis.com/openimages/web/index.html) on GPU server.
+
+<img src="../_static/openimages_v4.png" width="600">
+
+## Preparation
+
+The Open Images V4 dataset is available from official website, but original dataset is too large for the tutorial. We provide reduced dataset.
+
+Blueoil supports 2 formats for object detection.
+
+- OpenImagev4 format
+- DeLTA-Mark format
+
+Note: *Please see the detail in <a href="../usage/dataset.html">Prepare training dataset</a>*
+
+You can download subset of Open Images V4 from
+[our server](https://s3-ap-northeast-1.amazonaws.com/leapmind-public-storage/datasets/openimages.tgz).
+
+
+	$ wget https://s3-ap-northeast-1.amazonaws.com/leapmind-public-storage/datasets/openimages.tgz
+	$ tar xzf openimages.tgz
+
+
+This dataset consists of 5,000 color images in 2 classes, with 2,500 images per class.
+
+## Generate a configuration file
+
+Generate your model configuration file interactively by running `blueoil init` command.
+
+    $ ./blueoil.sh init
+
+This is an example of configuration.
+
+```
+#### Generate config ####
+your model name ():  objectdetection
+choose task type  object_detection
+choose network  LMFYoloQuantize
+choose dataset format  OpenImagesV4
+training dataset path:  {dataset_dir}
+set validataion dataset? (if answer no, the dataset will be separated for training and validation by 9:1 ratio.)  no
+batch size (integer):  16
+image size (integer x integer):  128x128
+how many epochs do you run training (integer):  100
+initial learning rate:  0.001
+choose learning rate setting(tune1 / tune2 / tune3 / fixed):  tune1 -> "2 times decay": tune1
+apply quantization at the first layer?  yes
+```
+
+## Train a neural network
+
+You can do it like 
```

Comment:
Why you not write the `$ ./blueoil.sh train config/{Model name}.yml` command here? Do you hesitate duplicate?
Also I think It is better the link is `./image_cls.html#train-a-neural-network` more than `./image_cls.html`.